Cargo.toml

```
@@ -20,3 +20,6 @@ members = [
 
 [profile.dev]
 opt-level = 2
+
+[patch.crates-io]
+sha2-asm = { git = "https://github.com/dignifiedquire/asm-hashes", branch = "arm-m1" }
```

needs to be removed once this is merged and released

---

Now that https://github.com/RustCrypto/asm-hashes/pull/35 is merged it can be changed to at least:

```suggestion
sha2-asm = { git = "https://github.com/RustCrypto/asm-hashes" }
```